pkg/translator/zipkin/zipkinv1/thrift.go

```diff
@@ -20,16 +20,14 @@ import (
 	"encoding/binary"
 	"errors"
 	"fmt"
+	"go.opentelemetry.io/collector/pdata/pcommon"
```

Comment:
This import should be moved down

Reply:
:) I did the PR from my personal laptop where I don't have the auto-format enabled, sorry.